CMakeLists.txt

```diff
@@ -35,14 +35,8 @@ endif()
 if(NBL_STATIC_BUILD)
 	message(STATUS "Static Nabla build enabled!")
 else()
-	if(MSVC)
-		if(NBL_DYNAMIC_MSVC_RUNTIME)
-			message(STATUS "Shared Nabla build enabled!")
-		else()
-			message(FATAL_ERROR "Turn NBL_DYNAMIC_MSVC_RUNTIME on! For dynamic Nabla builds dynamic MSVC runtime is mandatory!")
-		endif()
-	else()
-		message(FATAL_ERROR "Nabla can't be built with shared libraries! Please make sure you are targetting Windows OS and MSVC compiler!")
+	if(NOT NBL_COMPILER_DYNAMIC_RUNTIME)
+		message(FATAL_ERROR "Turn NBL_COMPILER_DYNAMIC_RUNTIME on! For dynamic Nabla builds dynamic runtime is mandatory!")
 	endif()
```

Comment:
aah ok so we want to ban compiling with `static-libstdc++`

Reply:
yes we agreed it must match